examples/custom_struct/packed_matrix.py

```diff
@@ -0,0 +1,91 @@
+import sys
+import numpy
+import cupy
+
+code = '''
+template<typename T>
+struct Matrix {
+    T value[4][4];
+
+    __device__ T& operator() (int i, int j) {
+        return this->value[i][j];
+    }
+
+    __device__ const T& operator() (int i, int j) const {
+        return this->value[i][j];
+    }
+};
+
+template<typename T>
+__device__ Matrix<T> operator+ (const Matrix<T>& lhs, const Matrix<T>& rhs) {
+    Matrix<T> res;
+    for (int i = 0; i<4; i++) {
+        for (int j = 0; j<4; j++) {
+            res(i,j) = lhs(i,j) + rhs(i,j);
+        }
+    }
+    return res;
+}
+
+template<typename T>
+__device__ Matrix<T> operator* (const Matrix<T>& lhs, const Matrix<T>& rhs) {
+    Matrix<T> res;
+    for (int i = 0; i<4; i++) {
+        for (int j = 0; j<4; j++) {
+            res(i,j) = T(0);
+            for (int k = 0; k<4; k++) {
+                res(i,j) += lhs(i,k) * rhs(k,j);
+            }
+        }
+    }
+    return res;
+}
+
+template<typename T>
+__global__ void kernel(const Matrix<T>* A,
+                       const Matrix<T>* B,
+                       const Matrix<T>  C,
+                             Matrix<T>* out) {
+  int i = threadIdx.x;
+  out[i] = A[i] * B[i] + C;
+}
+'''
+
+
+def main():
+    N = 8
+    module = cupy.RawModule(code=code, options=('-std=c++03',),
```

Comment:
Apparently we hit the same issue in two different PRs...See https://github.com/cupy/cupy/pull/4922#issuecomment-803718555.
```suggestion
    module = cupy.RawModule(code=code, options=('-std=c++11',),
```

Reply:
I was not aware that C++03 was CUDA 11 specific, hope CUDA 12 will bring support for C++98 !